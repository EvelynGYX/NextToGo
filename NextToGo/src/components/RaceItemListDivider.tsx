import React from 'react';
import {StyleSheet, View} from 'react-native';

const RaceItemListDivider = () => {
  return <View style={styles.divider} />;
};

const styles = StyleSheet.create({
  divider: {
    height: 1,
    backgroundColor: 'gray',
    marginRight: 4,
  },
});

export default RaceItemListDivider;
