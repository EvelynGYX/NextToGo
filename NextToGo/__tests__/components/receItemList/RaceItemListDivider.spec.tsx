import React from 'react';
import renderer from 'react-test-renderer';
import RaceItemListDivider from '../../../src/components/raceItemList/RaceItemListDivider';

describe('RaceItemListDivider', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<RaceItemListDivider />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
