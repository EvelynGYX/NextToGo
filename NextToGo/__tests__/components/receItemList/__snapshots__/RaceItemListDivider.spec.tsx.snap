// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RaceItemListDivider renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "grey",
      "height": 1,
      "marginRight": 4,
    }
  }
/>
`;
